engine/provider.ftl

```diff
@@ -158,46 +158,6 @@
     [#return result]
 [/#function]
 
-[#function getPluginsFromLayers pluginState ]
```

Comment:
Where has this moved to? Couldn't see a reference to it anymore?

Reply:
I couldn't find any use for it - I think it got superceded when the installation  of providers moved to the input pipeline.

I removed it as unused code but can add back in if you think there is a need. I will need to adjust it to correctly deal with the layerState depending on whether it is call inside or outside the input pipeline

Reply:
All good didn't realise that it was subsumed. All good